src/pages/Explorer/Components/UpgradeModal.svelte

```diff
@@ -0,0 +1,20 @@
+<script>
+  import Dialog from 'webkit/ui/Dialog'
+
+  let closeDialog
+  const DialogPromise = {
+    resolve: () => console.log('resolve'),
+    reject: () => console.log('reject'),
+    locking: 0,
+  }
+</script>
+
+<Dialog {DialogPromise} bind:closeDialog>
```

Comment:
Pass `$$props` to the dialog component as it's done [here](https://github.com/santiment/insights-app/blob/a2c74e0995beb8d242cee57708560384dea6e951/src/components/ShareDialog.svelte#L58).